my-sms/app/controllers/students_controller.rb

```diff
@@ -84,4 +87,10 @@ def destroy
   def find_student
     @student = Student.find(params[:id])
   end
+
+  def flash_errors
+    student.errors.full_messages.each do |error|
+      (flash[:error] ||= []) << error
+    end if student&.errors
```

Comment:
You can chain the "safe navigation" operator which would remove the need for the trailing `if student&.errors`,

```suggestion
    student&.errors&.full_messages&.each do |error|
      (flash[:error] ||= []) << error
    end
```

I do think the chaining looks a bit ugly though. My personal preference is short circuiting (return early from the method) although this opinion may vary project to project

```ruby
return unless student&.errors
student.errors.full_messages.each do |error|
...
end

Reply:
Didn't know about the return unless syntax.
Switched to that in 640d900caff9530e476b4857a1e4842cc82e001d

Reply:
Also I added some specs for this in 21acf9bd1752fa314e6f2e13f90c233c4e8a6543